src/api/schemas/user-schema.ts

```diff
@@ -99,6 +105,22 @@ const getUserById = object({
 
 const getUserActivityList = getUserById;
 
+const userAddFavoriteActivitySchema = object({
+    body: object({
+        //  Identifier attribute
+        identifier: object({
+            firebase_id: firebase_id
+        }).strict('Identifier field contains an invalid key'),
+
+        //  Update attribute
+        update: object({
+            favoriteActivities: favoriteActivities.optional()
+        })
+            .strict('Update field contains an invalid key')
+            .refine(({ favoriteActivities }) => favoriteActivities !== undefined, { message: 'favoriteActivities field must be defined' })
```

Comment:
not sure what this is for. if you need the `favoriteActivities` to be there all the time, remove `.optional()` from line 117 and remove this refine

Reply:
done